feat: adding section for migrations (#369)
Basis-Theory/developers.basistheory.com: docs/api/deprecations.mdx

| Reactor Formulas                                                    | October 16, 2022  | February 14, 2024              | Reactor Formulas have been deprecated in favor of Reactors with `code`. For more information, see the section below on [Migrating from Reactor Formulas](#migrating-from-reactor-formulas).                                                                                                                                                                                                                     |
| Application Key Regenerate                                          | April 17, 2024    | July 17, 2024                  | Regenerating API Keys for an Application has been replaced with [Application Keys](/docs/api/applications/application-keys), to Migrate you will Create a new Application Key and remove the old key. |

## Application Key Regenerate Deprecation

> Regenerating API Keys on an Application use to be done by calling the `Regenerate` endpoint on the Application. With the introduction of Application keys, this functionality has been replaced with the ability to create new Application Keys and remove the old key.

Reviewer: ```suggestion
Regenerating an application's API key used to involve calling the `Regenerate` endpoint on the Application. With the introduction of Application keys, this functionality has been replaced with the ability to create new Application Keys and remove the old key.
```
